### PR TITLE
Switch to gradle

### PR DESCRIPTION
**Pull request by hakanrw**
+ Removed SBT build files and integrated Gradle
+ Added new cut/paste functionality
+ Added new higher resolution icons

Test:
+ `./gradlew build` and `./gradlew run` executed without problems

### CODE REVIEW

**Comment by hakanrw** [comment]
@MehmetCandar the project should now work normally on Windows.

Tested on Windows 11 & Fedora 39 on Java 17

File: app/src/main/java/dev/candar/swing/JFFile.java [hunk 1]
```diff
@@ -28,9 +29,10 @@ class JFFile extends JPanel {
         gc.weightx = 1;
         gc.fill = GridBagConstraints.HORIZONTAL;
 
-        icon = new JLabel( 
-            new ImageIcon(Utils.iconToImage(isDirectory ? Resources.dirIcon : Resources.fileIcon)
-                .getScaledInstance(50, 50, Image.SCALE_FAST))
+        JLabel icon = new JLabel( 
+            (dim ?
```
Comment (MehmetCandar) [inline app/src/main/java/dev/candar/swing/JFFile.java:33]: Nitpick: Multiple line ternary operators are not preferred. Using if statement here would be better for readability.

**Review by MehmetCandar — APPROVED** [review]
Very well written. Good job!

File: app/src/main/java/dev/candar/swing/JFFile.java [hunk 2]
```diff
@@ -30,17 +29,26 @@ class JFFile extends JPanel {
...
 
-        add(new JLabel( 
-            new ImageIcon(Utils.iconToImage(isDirectory ? Resources.dirIcon : Resources.fileIcon)
-                .getScaledInstance(50, 50, Image.SCALE_FAST))
-        ), gc);
+        JLabel icon = new JLabel( 
+            (dim ?
+                isDirectory ? Resources.folderCutImageIcon : Resources.fileCutImageIcon :
+                isDirectory ? Resources.folderImageIcon : Resources.fileImageIcon)
+        );
```
Comment (MehmetCandar) [inline app/src/main/java/dev/candar/swing/JFFile.java:36]: Nitpick: Multiple line ternary operators are not preferred. Using if-else statements would be better for readability.

**Review by MehmetCandar — APPROVED** [review]
LGTM!

**Pull request merged**